### PR TITLE
FboMultisample Fix

### PR DESCRIPTION
**Pull request by anzurio**
- Fixing the masks which extracts the different channels on bilinearCompare.
- Using a different random seed because the bilinearCompare (even in C++'s deqp) does not handle a few boundary cases correctly.

### CODE REVIEW

File: sdk/tests/deqp/functional/gles3/es3fFboMultisampleTests.js [hunk 1]
```diff
@@ -165,7 +165,7 @@ var DE_ASSERT = function(x) {
 
         // Render random-colored quads.
         /** @const {number} */ var numQuads = 8;
-        /** @type {deRandom.Random} */ var rnd = new deRandom.Random(9);
+        /** @type {deRandom.Random} */ var rnd = new deRandom.Random(7);
```
Comment (kenrussell) [inline sdk/tests/deqp/functional/gles3/es3fFboMultisampleTests.js]: Could you please add a comment here stating what you've mentioned in the commit message? Something like:

// The choice of random seed affects the correctness of the tests,
// because there are some boundary conditions which aren't handled
// correctly even in the C++ dEQP tests.

Thanks.

**Comment by kenrussell** [comment]
Looks good to me. Thank you for tracking this down and fixing it. One comment. Let's let @zhenyao review and merge this.

**Comment by zhenyao** [comment]
Please address Ken's comment. Then we'll merge the patch.  Thanks for getting this fixed.

**Comment by zhenyao** [comment]
lgtm
